receiver/k8seventsreceiver/go.mod

```diff
@@ -3,43 +3,48 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8seve
 go 1.17
 
 require (
-	github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8sconfig v0.41.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/k8sconfig v0.37.1
```

Comment:
That doesn't look right.

Reply:
Right. Made the change. I think it can be resolved now. Looking into other review comments. Thanks.